### DISCUSSION TITLE
Laravel Bagisto ecommerce kurulumu

### DISCUSSION THREAD

**Discussion by aghaeian** [mewebstudio/pos#242] [Q&A]
Merhaba,
laravel 10 üzreinde olan bagisto projesinde bunu kurmak istedim ama ödeme yöntemlerinde gözükmüyor.

**Comment by nuryagdym** [comment]
Merhabalar, bagisto hakkinda bir bilgim yok. 
Laravel integrasyonu icin https://github.com/mewebstudio/laravel-pos bu kutuphane var ama ne kadar isinize yarayacak bilmiyorum.

Entegre edebilirseniz, ornek kodlari paylasirsaniz baska bagisto kullanicilar icin faydali olur.

**Comment by aghaeian** [comment]
Merhaba,
Yanıtınız içi teşekkür ederim. Bu dökümanıda kullandım ondada aynı sorun yaşanıyor. 
Farklı bir çalışma üzerinde papara için çalışma yapıtım ama sizin çalışma daha kapsamlı olduğu için kurmak istedim. 
Bu link benim çalışmam:
https://github.com/aghaeian/papara-payment-gateway.git

**Comment by nuryagdym** [comment]
konu ile alakasiz ama, elinde varsa papara dokumentasyonunu ve varsa ornek kodlari gonderir misin?

Kutuphaneye entegre ederim belki.

**Comment by aghaeian** [comment]
Merhaba Teşekkür ederim. Ekte dökümanı paylaşıyorum. İyi günler,nuryagdym ***@***.***> şunları yazdı (19 Eki 2024 18:29):﻿
konu ile alakasiz ama, elinde varsa papara dokumentasyonunu ve varsa ornek kodlari gonderir misin?
Kutuphaneye entegre ederim belki.

**Comment by aghaeian** [comment]
İyi günler,nuryagdym ***@***.***> şunları yazdı (19 Eki 2024 18:29):﻿
konu ile alakasiz ama, elinde varsa papara dokumentasyonunu ve varsa ornek kodlari gonderir misin?
Kutuphaneye entegre ederim belki.
